vms/proposervm/vm.go

```diff
@@ -220,6 +227,34 @@ func (vm *VM) Initialize(
 	default:
 		return err
 	}
+
+	vm.proposerBuildSlotGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "block_building_slot",
+			Help: "the post-durango slot in which the block was built",
```

Comment:
```suggestion
			Help: "the slot that this node may attempt to build a block",
```

Reply:
To clarify, everything is post-durango at this point. So it probably isn't meaningful to specify it in ongoing metrics.